packages/suite/src/components/wallet/TransactionItem/components/TargetAddressLabel.tsx

```diff
@@ -28,7 +35,7 @@ export const TargetAddressLabel = ({ target, type, accountMetadata }: TargetAddr
     }
 
     return (
-        <TruncatedSpan>
+        <TruncatedSpan isBlurred={isZeroValuePhishing}>
```

Comment:
This does not apply, the relevant transaction items use this component instead: `TokenTransferAddressLabel`.

Reply:
[743062e](https://github.com/trezor/trezor-suite/pull/7391/commits/743062eb2a78766ab992206c2d807d843b570601)